authbridge/demos/finance-sparc/scripts/show-verdicts.py

```
@@ -1,26 +1,53 @@
+each SPARC verdict. Pass the session-API base URL as the first argument
+(default http://localhost:19094).
+"""
 import json
 import sys
+import urllib.request
+
+BASE = sys.argv[1] if len(sys.argv) > 1 else "http://localhost:19094"
+
+
+def get(path):
+    with urllib.request.urlopen(BASE + path, timeout=10) as resp:
```

Ruff (0.15.15)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)